TROUBLESHOOTING.md

```diff
@@ -83,3 +83,30 @@ Try resetting the ownership by running:
 ```sh
 $ sudo chown -R <user> $HOME/.balena
 ```
+
+## Broken line wrapping / cursor behavior with `balena ssh`
+
+Users sometimes come across broken line wrapping / cursor behavior in text terminals, for example when long command lines are typed in a `balena ssh` session, or a misbehaving cursor when using text editors like `vim` or `nano`. As far as we know, this is not something specific to balena CLI, being also commonly reported issues with standard remote terminal tools like `ssh` or `telnet`. It is often a remote shell configuration issue (files like `/etc/profile`, `~/.bash_profile`, `~/.bash_login`, `~/.profile`, etc., or the lack of any such file at all), including UTF-8 misconfiguration, the use of ASCII control characters in shell prompt formatting (e.g. `$PS1` env var) or the output of tools or log files that use colored text. The issue can sometimes be fixed by resizing the client terminal window, or by running one or more of the following commands on the shell:
```

Comment:
Probably worth removing the 'As far as we know' line, as we then describe why it usually isn't.

Reply:
Done